app/models/conference_preference.rb

```diff
@@ -1,5 +1,6 @@
 class ConferencePreference < ActiveRecord::Base
 
-  belongs_to :team, inverse_of: :conference_preferences
-  belongs_to :conference, inverse_of: :conference_preferences
-end
+  belongs_to :team, inverse_of: :conference_preference
+  belongs_to :first_conference, class_name: 'Conference'
```

Comment:
Again personal preference, but I'd simply name this:
```ruby
belongs_to :conference1
belongs_to :conference2
```
Semi aligned with how it's done for the project choices for teams and with the idea in mind to not indicate a) both needing to be present and b) that teams to to two conferences _(first to the first one, then to the second one)_